stl/inc/charconv

```diff
@@ -824,54 +761,6 @@ _NODISCARD inline bool _Multiply_by_power_of_ten(_Big_integer_flt& _Xval, const
     return _Multiply(_Xval, _Small_powers_of_ten[_Small_power - 1]); // when overflow occurs, resets to zero
 }
 
-// The following non-compiled code is the generator for the big powers of ten table found in

```

Comment:
Generator programs were moved elsewhere. 

If STL `.cpp` files are built with `/std:c++20` or `/std:c++latest`, the programs can probably be converted to `consteval` functions